docs/getting-started.md

```diff
@@ -134,6 +134,12 @@ npx playwright test --trace=on # get a time machine attached to each test result
 # remove docker image when done
 npm run docker:stop
 ```
+If you want to run tests on a local build you will need to update your secrets file to include this line.
+
+```bash
```

Comment:
Elsewhere in docs we've been using the `dotnet user-secrets` tool to set secrets - this is to stop someone from misunderstanding and accidentally putting secrets in appsettings! 
```
dotnet user-secrets set "TestOverride:PlaywrightTestSecret" "TestSuperSecret"
```